posts/2022-02-21-rust-analyzer-joins-rust-org.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: "rust-analyzer joins the Rust organization!"
+author: The rust-analyzer Team on behalf of the entire Rust Team
+---
+
+We have an exciting announcement to make!
+The [rust-analyzer](https://rust-analyzer.github.io) project, a new implementation of the Language Server Protocol (LSP) for Rust, is now officially a part of the wider Rust organization! :tada:
+
+We want to start by thanking everyone who has gotten us this far, from contributors, to sponsors, to all the users of rust-analyzer in the Rust community. We could not have done this without you.
+
+The immediate impact of this organizational change is limited -- nothing changes for rust-analyzer users or contributors.
+However, this change unblocks technical work to make rust-analyzer the officially recommended language server for Rust in the near future.
+
+If you were hesitant to try rust-analyzer before, today is a good opportunity to do so.
+Not only is it a very capable language server for Rust, but according to [VS Code statistics](https://marketplace.visualstudio.com/search?target=VSCode&category=Programming%20Languages&sortBy=Rating), it is one of the best rated LSP implementations *across* programming languages.
+We highly recommend giving rust-analyzer a spin today, even if it will take some more time for us to complete the due process and switch from the existing officially recommended LSP implementation (RLS) properly.
+
+rust-analyzer enjoys excellent support in many editors:
+
+* For VS Code, install the [rust-analyzer](https://marketplace.visualstudio.com/items?itemName=matklad.rust-analyzer) extension from the marketplace.
+* For Neovim, follow the setup instructions [from this post](https://sharksforarms.dev/posts/neovim-rust/).
+  For Vim, see [coc-rust-analyzer](https://github.com/fannheyward/coc-rust-analyzer).
+* For Emacs, follow the setup instructions [from this post](https://robert.kra.hn/posts/2021-02-07_rust-with-emacs/).
+
+For other editors, check [the manual](https://rust-analyzer.github.io/manual.html).
+Finally, if you are using IntelliJ-Platform based IDEs like CLion, IDEA or PyCharm, you don't need rust-analyzer.
+You should use the awesome [IntelliJ Rust](https://intellij-rust.github.io) plugin by JetBrains.
+
+## History and Future
+
+The rust-analyzer project was started at the very end of 2017 ([first commit](https://github.com/rust-analyzer/rust-analyzer/commit/a63222cd240d9b5405826783603f3b391c90885d)).
+At that time, the existing LSP implementation, RLS, had been providing IDE support for Rust for several years.
+While it handled some of most important and essential features, it was clearly far from state-of-the art in other languages' ecosystems.
```

Comment:
I found the second half of the sentence a little confusing, would something like this be more clear? (or perhaps I've just misunderstood?)

```suggestion
While it handled some of most important and essential features, it was clearly far from the state-of-the art experience found in some other languages' ecosystems.
```

Reply:
That looks like a good change. But also, "some of most" -> "some of the most".

```suggestion
While it handled some of the most important and essential features, it was clearly far from the state-of-the art experience found in some other languages' ecosystems.
```

Reply:
Picked @lnicola version for now, open to bikeshed!